src/index.html

```diff
@@ -1,14 +1,40 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container container">
+        <a href="#home" class="header__logo">
+          <img src="images/logo.png" alt="Moyo-logo">
+        </a>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#!" class="nav__link is-active">Apple</a></li>
+            <li class="nav__item"><a href="#!" class="nav__link">Samsung</a></li>
+            <li class="nav__item"><a href="#!" class="nav__link">Smartphones</a></li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#!" class="nav__link">
                Smartphones
              </a>
            </li>
            
```
fix code style all over the page